pkg/aws/s3/connect.go

```diff
@@ -91,3 +91,32 @@ func GetAWSSession(vaultToken, region, accountLocation string) (*session.Session
 	session := session.Must(newSess, err)
 	return session, nil
 }
+
+func GetAWSSecretKey(vaultToken, accountLocation string) (string, string, error) {
+	vc, err := vault.GetVaultClientByToken(vaultToken)
+	if err != nil {
+		return "", "", err
+	}
+
+	secretPath := ""
+	if accountLocation == US_VAULT {
+		secretPath = sharedAws.US_VAULT_SECRET_PATH
+	} else if accountLocation == INDIA_VAULT {
+		secretPath = sharedAws.INDIA_VAULT_SECRET_PATH
+	} else {
+		return "", "", fmt.Errorf("invalid AWS account location provided : %s", accountLocation)
```

Comment:
We don't need this separation anymore. There is only an INDIA_VAULT now

Reply:
should we not keep it for future use cases?

Reply:
No not needed, that account is deprecated.
We can keep it only if it doesn't harm management/not causes confusion